HRTool/src/main/webapp/index.html

```diff
@@ -1,82 +1,34 @@
 <!DOCTYPE html>
-<html ng-app="app">
-<head>
-<title>Login page</title>
-<!-- Latest compiled and minified CSS 
-<link rel="stylesheet"
-	href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
-
-<!-- jQuery library 
-<script
-	src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script> 
-
-
-<script
-	src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>-->
-
-	  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
-	  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.min.js"></script>
-	  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-route.min.js"></script>
-	  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-aria.min.js"></script>
-	<script
-	src="https://cdnjs.cloudflare.com/ajax/libs/angular-material/0.10.0/angular-material.js"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-messages.min.js"></script>
-
-	<meta name="google-signin-scope" content="profile email">
-	<meta name="google-signin-client_id" content="282087479252-4v31a07nrjnmfganchk4i1btpvoprjro.apps.googleusercontent.com">
-    <script src="https://apis.google.com/js/platform.js" async defer></script>
-
-	<script type="text/javascript" src="https://rawgit.com/betsol/angular-input-date/master/src/angular-input-date.js"></script>
-<!--<script src="app/js/jquery.min.js"></script>-->
-	<script src="app/app.js"></script>
-	<link rel="stylesheet" type="text/css" href="app/css/css.css">
-<!-- concatenated flow.js + ng-flow libraries -->
-	<script src="app/js//ng-flow-standalone.min.js"></script>
-<!-- or include the files separately -->
-	<script src="app/js/ng-flow.js"></script>
-	<script src="app/js/ng-flow.min.js"></script>
+<html>

```

Comment:
This is exactly as it should be. :)
Some convention is to leave both head and body tags un-indented.